imports/components/cards/cards.Activity.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import { Component } from "react";
+import styled from "styled-components";
+import moment from "moment";
+import { Link } from "react-router";
+import { Error, Success } from "../icons";
+
+const Paragraph = styled.p`
+  font-family: colfax-web, sans-serif;
+`;
+
+const BoldedDate = styled(Paragraph)`
+  font-weight: bold;
+  display: inline-block;
+  vertical-align: top;
+`;
+
+const Strong = styled.strong`
+  font-family: colfax-web, sans-serif;
+`;
+
+type IActivity = {
+  status: string,
+  transaction: Object,
```

Comment:
I don't want us to force this card to be used with just transactions. Instead, I think we should setup the props as:

```js
type IActivity = {
  status: string,
  date: ?string,
  description: string,
  linkText: ?string,
  linkUrl: ?string,
};

```

Then check existence of the link and date props to see if we need to render them